src/compass/version.py

```diff
@@ -8,6 +8,7 @@
 # release history
 Tag = collections.namedtuple('Tag', 'version date')
 release_history = (
+    Tag('0.5.0', '2023-08-25'),
```

Comment:
@vbrancat I originally intended this PR to take care of the Docker only, but I think it would be more efficient to to extend its scope to prepare for the final delivery. As part of the preparation, I also included the change to bump the version. Please let me know if you don'y like the version number. Thanks.

Reply:
I have no problem with the version number.